helm/templates/api/api-configmap.yaml

```diff
@@ -396,7 +396,16 @@ data:
       {{- end }}
       # Allows to define if cookie secure only (default false)
       cookie-secure: {{ .Values.api.jwt.cookie.secure | default false }}
-
+    security:
+      defaultAdmin: {{ .Values.api.security.defaultAdmin | default true }}
+      accountAccessTokens:
+        encoder:
+          settings:
+            rounds: {{ .Values.api.security.accountTokenBcryptRounds | default 10 }}
+      {{- if .Values.api.security.providers }}
+      providers:
+        {{- toYaml .Values.api.security.providers | nindent 8 }}
+      {{- end}}
```

Comment:
suggestion: I would do a test on the **security** section as you did for `.Values.api.security.providers` line 405. In that way we are expecting the same structure as the one in the gravitee.yaml. And in future if new entries in the security section are added, they will be taken into account automatically. wdyt ?